stdlib/os/__init__.pyi

```diff
@@ -822,6 +822,34 @@ else:
 
 if sys.platform != "win32":
     from posix import sched_param
+
+    if sys.version_info >= (3, 8):
+        def posix_spawn(
+            path: _FdOrAnyPath,
+            argv: _ExecVArgs,
+            env: _ExecEnv,
+            *,
+            file_actions: Optional[Sequence[Tuple[int, Any]]] = ...,
```

Comment:
Looks like this can have more than two items in the tuple: https://github.com/python/cpython/blob/main/Modules/posixmodule.c#L6052. 
```suggestion
            file_actions: Sequence[Tuple[Any, ...]] | None = ...,
```